src/main/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerChildListValidator.java

```diff
@@ -54,12 +54,14 @@ public boolean isValidLocations(CaseWorkerProfile caseWorkerProfile, ConstraintV
         if (isNotEmpty(caseWorkerProfile.getLocations())
             && caseWorkerProfile.getLocations().size() > 1) {
             //@TO do remove getLocationName with Id problem with excel sheet
-            isValidLocations = negate(caseWorkerProfile.getLocations().get(0).getLocationName()
-                .equalsIgnoreCase(caseWorkerProfile.getLocations().get(1).getLocationName()));
-            if (FALSE.equals(isValidLocations)) {
-                context.buildConstraintViolationWithTemplate(DUPLICATE_PRIMARY_AND_SECONDARY_LOCATIONS)
-                    .addPropertyNode(LOCATION_FIELD)
-                    .addConstraintViolation();
+            if (caseWorkerProfile.getLocations().get(0).getLocationName() != null) {
+                isValidLocations = negate(caseWorkerProfile.getLocations().get(0).getLocationName()
```

Comment:
can we use ObjectUtils.nullSafeEquals(caseWorkerProfile.getLocations().get(0).getLocationName(), caseWorkerProfile.getLocations().get(1).getLocationName()) so we can avoid the null checks in our code

**org.springframework.util.ObjectUtils**

Reply:
Sure, I'll make the change